requirement.txt

```diff
@@ -0,0 +1,3 @@
+plotly==3.10.0
```

Comment:
file is normally called "requirements.txt". Note the 's'.

Because you have dependencies you should also have a README or INSTALL file telling the user how to use this.
Feel free to do in later cl.